NativeScript/runtime/Runtime.mm

```diff
@@ -176,6 +178,80 @@ void DisposeIsolateWhenPossible(Isolate* isolate) {
     PromiseProxy::Init(context);
     Console::Init(context);
     WeakRef::Init(context);
+    
+    
+    auto blob_methods = R"js(
+    const BLOB_STORE = new Map();
+    URL.createObjectURL = function (object, options = null) {
+        try {
+            if (object instanceof Blob || object instanceof File) {
+                const id = NSUUID.UUID().UUIDString;
```

Comment:
@triniwiz you need to convert the UUID to lowercase. iOS generates UUID in uppercase but the standard is to use lower case:

> Are UUIDs Case-sensitive?
> No, UUIDs are written in base 16 which uses numbers 0-9 and characters a-f. There is no distinction between upper and lowercase letters. However, [RCF 4122 section 3](https://tools.ietf.org/html/rfc4122#section-3) requires that UUID generators output in lowercase and systems accept UUIDs in upper and lowercase.